### ISSUE TITLE
Failing Build in AFNI>21.2.01

### ISSUE THREAD

**Issue by liamtimms** [afni/afni#330]
Hi, I'm currently maintaining an unofficial Arch Linux User Repositiory package for AFNI. This is admittedly outside the score of officially supported platforms for AFNI. That said, the community has been able to keep the build working since 2016 (https://aur.archlinux.org/packages/afni/#news).

I'm currently trying to update the package to the recent release. This following build error has cropped up in the lastest versions of AFNI (while older versions are still buildable):

```
/usr/bin/ld: ./libcoxplot.a(pwritf.o):(.bss+0x80): multiple definition of `zzzplt_'; ./libcoxplot.a(ppak_commons.o):(.bss+0xc0): first defined here
/usr/bin/ld: ./libcoxplot.a(pwritf.o):(.bss+0x0): multiple definition of `zzpltr_'; ./libcoxplot.a(ppak_commons.o):(.bss+0x40): first defined here
/usr/bin/ld: libXmHTML.a(LZWStream.o): in function `LZWStreamInit':
LZWStream.c:(.text+0xdd3): warning: the use of `tmpnam' is dangerous, better use `mkstemp'
collect2: error: ld returned 1 exit status
make: *** [Makefile.INCLUDE:1842: afni] Error 1
```

Here are the versions I have tested with a check indicating if the build successfully completed:

#### 21.1

- [x] 21.1.08
- [x] 21.1.15
- [x] 21.1.20

#### 21.2

- [x] 21.2.00
- [x] 21.2.01

then these versions all fail with the same error:

- [ ] 21.2.02
- [ ] 21.2.03
- [ ] 21.2.05
- [ ] 21.2.10

#### 21.3

- [ ] 21.3.01
- [ ] 21.3.14

In summary, this error seems to be consistently hitting builds starting at AFNI 21.2.02 and continuing from there. I'm not sure if if it makes sense to fix it upstream in AFNI itself or downstream in packaging with some patch. While this might not strictly be a bug for AFNI itself, any help to solving this or pointing me the right direction would be appreciated.

**Comment by afni-rickr** [comment]
Hi @liamtimms.  The C files in that coxplot directory are actually converted from fortran, and tend to need clean-up after the fact.  It looks like between those 2 tags one such C file was regenerated and still needs some emotional support.  I will take a closer look.  Thanks for bringing this up!

**Comment by afni-rickr** [comment]
Hi @liamtimms, are you able to try pull request #331, or does it need to be in master?  Thanks!

**Comment by liamtimms** [comment]
Hi @afni-rickr , I used that PR as a patch on the 21.3.16 release and it seems to have built successfully! I'll wait for a new release tag with it merged into master to update the package for the community but that PR does seem to solve the issue. Thanks!

**Comment by afni-rickr** [comment]
@liamtimms Great, I will fire off a build tonight, thanks!

**Comment by liamtimms** [comment]
@mrneont seems to be yes! Thank you!
